apps/web/src/containers/NoteView/note-supply/note-summary.tsx

```diff
@@ -16,23 +16,38 @@ import {
   H5,
   NoteChart,
   H2,
+  DateRangeButtons,
+  dateRangeData,
```

Comment:
This shouldn't be in the MUI package in my opinion. Also it should be named something more descriptive like `ValidDateRanges`.

Reply:
I put it in mui since I figured we would probably want to use is with other charts in the app. Where do you purpose I put it instead? Shared web?